x-pack/plugins/ml/public/application/model_management/delete_models_modal.tsx

```diff
@@ -44,6 +45,14 @@ export const DeleteModelsModal: FC<DeleteModelsModalProps> = ({ models, onClose
     WithRequired<ModelItem, 'pipelines'>
   >;
 
+  const modelsWithInferenceAPIs = models.filter(
+    (m) => Array.isArray(m.inference_apis) && m.inference_apis.length > 0
```

Comment:
This should be using `hasInferenceServices`

Reply:
Updated https://github.com/elastic/kibana/pull/175880/commits/4156b03faa88ada2cfc27e3a81e5619b91b0eb57